content/momentum/4/using-dkim.md

```diff
@@ -75,6 +76,8 @@ All DKIM verification implementations must support key sizes of 512, 768, 1024,
 
 Note that Google requires all senders to sign with a 1024 bit or greater DKIM key size.
 
+It is recommended that a key size of at least 2048 bits is used.
```

Comment:
This is from a proposed standard (RFC 8301) that BTW also suggests that the "must support" range is 1024 - 4096.